core/feature_flag_list.py

```diff
@@ -111,6 +110,7 @@ class FeatureNames(enum.Enum):
     FeatureNames.IS_IMPROVEMENTS_TAB_ENABLED,
     FeatureNames.LEARNER_GROUPS_ARE_ENABLED,
     FeatureNames.ENABLE_VOICEOVER_CONTRIBUTION,
+    FeatureNames.AUTO_UPDATE_EXP_VOICE_ARTIST_LINK,
```

Comment:
This flag helps to automatically update the model (i.e., linking exploration & voice artist) after the migration job has been run.
This feature is tested and it is safe to move the flag to PROD.

Reply:
Sounds good.